src/dsql/parse.y

```diff
@@ -8682,6 +8804,21 @@ non_reserved_word
 	| TOTALORDER
 	| TRAPS
 	| ZONE
+	| MODE				// crypt functions
```

Comment:
@AlexPeshkoff on this area every keyword should be listed alphabetically for the FB version in general, not per feature.